app/controllers/incidents_controller.rb

```diff
@@ -131,6 +133,8 @@ def unclaimed
   end
 
   def update
+    @reason_codes = ReasonCode.order(:identifier)
+    @supplementary_reason_codes = SupplementaryReasonCode.order(:identifier)
```

Comment:
Why add these instance variables right here?

Reply:
As opposed to a `before_action`, you're saying? Or are you questioning something else?

Reply:
Just not sure if I'm missing something big--update actions don't usually respond to get requests so where are these used?

Reply:
The `else` renders a view that needs them.

Reply:
OMG I knew I was missing something. 👯‍♂️